libbz2-rs-sys/src/bzlib.rs

```diff
@@ -1317,7 +1317,7 @@ fn un_rle_obuf_to_output_fast(strm: &mut BzStream<DState>, s: &mut DState) -> bo
             BZ_RAND_UPD_MASK!(s);
             k1 ^= BZ_RAND_MASK!(s);
             s.nblock_used += 1;
-            s.state_out_len = k1 as i32 + 4;
+            s.state_out_len = k1 as u32 + 4;
```

Comment:
You can use `u32::from(k1)` to know for sure that this doesn't silently truncate.

Reply:
`k1` is a u8 so this will never cause issues